### ISSUE TITLE
Problem with newPage function

### ISSUE THREAD

**Issue by entrptaher** [casperjs/casperjs#1533]
This is the code copied from the doc. Just added .echo to be sure what it's doing. However anything after newPage is ignored. Probably the program exists. In my case, there are no extra tcp connections but the problem exists.

```js
var casper = require('casper').create();

casper.start('http://localhost/', function() {
    this.echo("Alpha");
});

casper.then(function() {
    casper.page = casper.newPage();
    casper.open('http://localhost/').then( function() {
       this.echo("Bita"); // Never gets printed
    });
});

casper.then(function(){
  this.echo("Gamma"); // Never gets printed
});

casper.run();
```

**Comment by istr** [comment]
Could you make that work using ```casper.thenOpen``` instead of ```casper.open``` like in the test case 
https://github.com/n1k0/casperjs/blob/master/tests/suites/casper/newpage.js?

If so we should update the documentation.

**Comment by entrptaher** [comment]
Yes I tried thenOpen too. it didn't work.
Sorry for not mentioning earlier.

**Comment by entrptaher** [comment]
Doesn't work

```js
casper.thenOpen('http://localhost/', function() {
       this.echo("Bita"); // Never gets printed
    });
```

Doesn't work either
```js
casper.thenOpen('http://localhost/').then( function() {
       this.echo("Bita"); // Never gets printed
    });
```

**Comment by entrptaher** [comment]
Now its even weird.
Just checked all casperjs files, not a single file named newpage.js
Checked casper module, not a single module found.

I am running casperjs 1.1.0.beta5 on windows through npm.
I searched the casperjs file which was not downloaded through npm, same problem.
I am missing something perhaps.

**Comment by entrptaher** [comment]
Ok, so problem found.

Whoever installed casperjs using npm or manually had 1.1.0.beta5 installed. And that tree doesn't have newpage.

https://github.com/n1k0/casperjs/tree/1.1-beta5/tests/suites/casper

Fix: Install version 1.1.0 which was released a day ago :/ .

**Comment by mickaelandrieu** [comment]
Thanks for report! Can you create a pull request opened  docs saying that newPage() is only available since 1.1.0?

**Comment by istr** [comment]
Hm... Sorry for inconvenience. :-(
Yes, a PR with an updated "available since" would be great. Missed to check the new features to match the docs.

**Comment by entrptaher** [comment]
It says new in version 1.1.
Should we just change the text to **Available since version 1.1.0** ?

**Comment by entrptaher** [comment]
c0b0fd0

**Comment by istr** [comment]
> Should we just change the text to Available since version 1.1.0 ?

That will do.

**Comment by istr** [comment]
> Should we just change the text to Available since version 1.1.0 ?

That will do.

**Comment by entrptaher** [comment]
Now I see,
So the commit wasn't approved and/merged?
Did I miss something?

**Comment by BIGjuevos** [comment]
It doesn't appear to be in official history. If you put up a PR I'll merge it for you.

**Comment by BIGjuevos** [comment]
this was closed by #1548
